.github/workflows/docker.yml

```diff
@@ -25,9 +24,9 @@ jobs:
         with:
           username: ${{ secrets.DOCKER_USERNAME }}
           password: ${{ secrets.DOCKER_PASSWORD }}
-      - name: Build container (latest)
-        if: ${{ github.ref == 'refs/heads/master' }}
-        run: 'docker buildx build --push -t ghostunnel/ghostunnel:latest --platform linux/amd64,linux/arm64,linux/arm/v7 .'
+#      - name: Build container (latest)
```

Comment:
just wondering what's the reasoning behind commenting this step out (making sure it's intentional)

Reply:
yeah I figured we don’t need a latest image, only the tagged one so I intentionally just commented this out.

Reply:
as long as nothing is dependent on that latest image, sgtm!